nautobot/docs/release-notes/version-1.3.md

```diff
@@ -140,9 +140,36 @@ We've updated the Jinja2 dependency from version 2.11 to version 3.0.3. This may
 
 As Python 3.6 has reached end-of-life, and many of Nautobot's dependencies have already dropped support for Python 3.6 as a consequence, Nautobot 1.3 and later do not support installation under Python 3.6.
 
+## v1.3.3 (2022-05-02)
+
+### Added
+
+- [#1481](https://github.com/nautobot/nautobot/issues/1481) - Pre-Generate Docs, Add Support for Plugin-Provided Docs
+- [#1617](https://github.com/nautobot/nautobot/pull/1617) - Added `run_job_for_testing` helper method for testing Jobs in plugins, internally.
+
+### Changed
+
+- [#1680](https://github.com/nautobot/nautobot/pull/1680) - Bump netutils dependency to 1.1.0.
```

Comment:
Possibly add an additional reference to #1481 here under "Changed", stating that the `docs` footer link now leads to the embedded docs rather than readthedocs?